.github/workflows/ci.yml

```diff
@@ -34,13 +34,70 @@ jobs:
           version: '0.1'
       - name: Run Cargo.toml linter
         run: git ls-files | grep Cargo.toml$ | xargs --verbose -n 1 cargo-toml-lint
+      
+  frontend:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        project: ["auctions/english-auction"]
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Install Fuel toolchain
+        uses: FuelLabs/action-fuel-toolchain@v0.2.0
+        with:
+          name: my-toolchain
+          components: forc@${{ env.FORC_VERSION }}, fuel-core@${{ env.CORE_VERSION }}
+
+      - name: Setup Node
+        uses: ./.github/actions/setup-node
+
+      - name: Setup Docker
+        uses: ./.github/actions/setup-docker
+        with:
+            password: ${{ secrets.GITHUB_TOKEN }}
+      
+      - name: Run Lint
+        run: |
+          cd ${{ matrix.project }}
+          pnpm install
+          pnpm build
+          pnpm lint
+
+      - name: Initialization
+        run: |
+          cd ${{ matrix.project }}
+          pnpm services:setup-test
+        
+      - name: Install Playwright
+        run: |
+          cd ${{ matrix.project }}
+          pnpm exec playwright install --with-deps chromium
+      
+      - name: Run E2E Tests
+        run: |
+          cd ${{ matrix.project }}
+          xvfb-run --auto-servernum -- pnpm test
+        env:
+          NODE_ENV: test
+
+      - name: Upload Test Results
```

Comment:
Upload?

Reply:
yeah we upload the playwright test results to github

Reply:
it's super useful if the tests are passing locally but failing on the ci